src/poetry/console/application.py

```diff
@@ -232,23 +232,27 @@ def register_command_loggers(
         handler = IOHandler(io)
         handler.setFormatter(IOFormatter())
 
+        level = logging.WARNING
+
+        if io.is_debug():
+            level = logging.DEBUG
+        elif io.is_very_verbose() or io.is_verbose():
+            level = logging.INFO
+
+        logging.basicConfig(level=level, handlers=[handler])
+
         for name in loggers:
             logger = logging.getLogger(name)
 
             logger.handlers = [handler]
 
-            level = logging.WARNING
+            _level = level
             # The builders loggers are special and we can actually
             # start at the INFO level.
             if logger.name.startswith("poetry.core.masonry.builders"):
-                level = logging.INFO
-
-            if io.is_debug():
-                level = logging.DEBUG
-            elif io.is_very_verbose() or io.is_verbose():
-                level = logging.INFO
+                _level = logging.INFO
```

Comment:
The log level for `"poetry.core.masonry.builders"` is always `INFO` now. Before it could also be `DEBUG`. Is this intended?

Reply:
Good catch fixing.